docs/media-analytics/custom-player.md

```diff
@@ -8,7 +8,7 @@ In this guide we will learn how you can implement [Media Analytics](http://www.m
  
 ## Tracking a new media player in the browser
 
-The Media Analytics plugin currently supports  out of the box the HTML5, Vimeo and YouTube media players. If you are using
+The Media Analytics plugin currently supports out of the box the HTML5, Vimeo and YouTube media players. If you are using
```

Comment:
Could we also here mention JW Player, VideoJS ? as people don't necessarily know that these players use html5